include/nvexec/stream/algorithm_base.cuh

```diff
@@ -102,47 +100,46 @@ namespace nvexec::STDEXEC_STREAM_DETAIL_NS::__algo_range_init_fun {
 
       template <class Range>
-      using set_value_t = typename DerivedSender::template set_value_t<Range>;
+      using _set_value_t = typename DerivedSender::template _set_value_t<Range>;
 
       Sender sndr_;
       STDEXEC_NO_UNIQUE_ADDRESS InitT init_;
       STDEXEC_NO_UNIQUE_ADDRESS Fun fun_;
 
       template <class Self, class Env>
       using completion_signatures = //
-        stdexec::make_completion_signatures<
-          stdexec::__copy_cvref_t<Self, Sender>,
+        make_completion_signatures<
+          __copy_cvref_t<Self, Sender>,
           Env,
-          stdexec::completion_signatures<stdexec::set_error_t(cudaError_t)>,
-          set_value_t >;
-
-      template <stdexec::__decays_to<__t> Self, stdexec::receiver Receiver>
-        requires stdexec::
-                   receiver_of<Receiver, completion_signatures<Self, stdexec::env_of_t<Receiver>>>
-      friend auto tag_invoke(stdexec::connect_t, Self&& self, Receiver rcvr)
-        -> stream_op_state_t< stdexec::__copy_cvref_t<Self, Sender>, receiver_t<Receiver>, Receiver> {
-        return stream_op_state<stdexec::__copy_cvref_t<Self, Sender>>(
-          ((Self &&) self).sndr_,
-          (Receiver &&) rcvr,
+          completion_signatures<set_error_t(cudaError_t)>,
+          _set_value_t >;
+
+      template <__decays_to<__t> Self, receiver Receiver>
+        requires receiver_of<Receiver, completion_signatures<Self, env_of_t<Receiver>>>
+      friend auto tag_invoke(connect_t, Self&& self, Receiver rcvr)
+        -> stream_op_state_t< __copy_cvref_t<Self, Sender>, receiver_t<Receiver>, Receiver> {
+        return stream_op_state<__copy_cvref_t<Self, Sender>>(
+          ((Self&&) self).sndr_,
+          (Receiver&&) rcvr,
           [&](operation_state_base_t<stdexec::__id<Receiver>>& stream_provider)
             -> receiver_t<Receiver> {
             return receiver_t<Receiver>(self.init_, self.fun_, stream_provider);
           });
       }
 
-      template <stdexec::__decays_to<__t> Self, class Env>
-      friend auto tag_invoke(stdexec::get_completion_signatures_t, Self&&, Env)
-        -> stdexec::dependent_completion_signatures<Env>;
+      template <__decays_to<__t> Self, class Env>
+      friend auto tag_invoke(get_completion_signatures_t, Self&&, Env)
+        -> dependent_completion_signatures<Env>;
 
-      template <stdexec::__decays_to<__t> Self, class Env>
-      friend auto tag_invoke(stdexec::get_completion_signatures_t, Self&&, Env)
+      template <__decays_to<__t> Self, class Env>
+      friend auto tag_invoke(get_completion_signatures_t, Self&&, Env)
         -> completion_signatures<Self, Env>
         requires true;
 
-      friend auto tag_invoke(stdexec::get_env_t, const __t& self) //
-        noexcept(stdexec::__nothrow_callable<stdexec::get_env_t, const Sender&>)
-          -> stdexec::__call_result_t<stdexec::get_env_t, const Sender&> {
-        return stdexec::get_env(self.sndr_);
```

Comment:
Isn't it here a good idea to call `stdexec::get_env(self.sndr_)` instead of `get_env(self.sndr_)` because of ADL ?

Reply:
`get_env` is an object, not a function. No ADL if it's found in phase one name lookup.